### ISSUE TITLE
Tutorial seems to be outdated

### ISSUE THREAD

**Issue by alexisrosuel** [zalando/expan#205]
Hi ! Thanks a lot for this awesome project! I tried to install expan with `pip install expan` and go through the tutorial given on readme.md of the github project homepage, but I have two errors:
 
``` 
>>> from expan.core.statistical_test import KPI, Variants, StatisticalTest
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'expan.core.statistical_test'
```

and 

``` 
>>> data, metadata = generate_random_data()
>>> exp = Experiment(data=data, metadata=metadata)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() missing 1 required positional argument: 'control_variant_name'
``` 

Is the last release not consistent? I use python 3.6 on mac os.

Thanks again !

**Comment by shansfolder** [comment]
Hello @alexisrosuel , thanks for using ExpAn!

The default page of doc is based on **master** branch, which haven't release yet. (our bad, shouldn't merge but not release). 
For your questions, you can select the version of expan you installed. e.g. if your expan is v.0.6.13: http://expan.readthedocs.io/en/v0.6.13/

Although there are a lot of changes in the latest one, and we will release it very soon.

**Comment by alexisrosuel** [comment]
You are so fast ;) Thanks, indeed it works !

**Comment by shansfolder** [comment]
@alexisrosuel We have released the newest v1.0.0. Try it out if you want to. (and docs are thus up to date now) ;)
